src/main/kotlin/no/nav/familie/klage/formkrav/FormService.kt

```diff
@@ -20,12 +22,14 @@ class FormService(
     private val stegService: StegService,
     private val behandlingService: BehandlingService,
     private val vurderingService: VurderingService
+    private val taskRepository: TaskRepository
 ) {
 
     fun hentForm(behandlingId: UUID): Form = formRepository.findByIdOrThrow(behandlingId)
 
     @Transactional
     fun opprettInitielleFormkrav(behandlingId: UUID): Form {
+        taskRepository.save(BehandlingsstatistikkTask.opprettPåbegyntTask(behandlingId=behandlingId))
```

Comment:
Hvorfor ligger denne her og ikke i opprettBehandling?

Reply:
I opprettBehandling så lages en task av typen "mottatt", og i formkrav så lages en av typen "påbegynt" ?

Reply:
Men denne kalles jo av opprettBehandling? Så først opprettese denne tasken, så opprettes den neste...? Virker litt ulogisk, eller?

<img width="624" alt="Skjermbilde 2022-10-28 kl  15 32 55" src="https://user-images.githubusercontent.com/402915/198613352-a2cea44d-8e79-491a-bba1-7cd4725b2413.png">

Reply:
Ja, sant, den bør kanskje flyttes til oppdaterForm ? Men at man lagrer tasken bare første gangen, siden man kan gå tilbake og oppdatere formen igjen ? Eller er det et annet sted som er bedre for hendelsen "påbegynt" ?